.github/renovate.json5

```diff
@@ -98,6 +105,20 @@
           "third_party/**"
       ],
       "enabled": false
+    },
+    {
+      // Only patch level updates for container images in the imagevector. Minor and major versions are updated manually.
+      "matchDatasources": ["docker"],
+      "matchUpdateTypes": ["major", "minor"],
+      "matchFileNames": ["imagevector\/images\\.yaml"],
+      "enabled": false
+    },
+    {
+      // Ignore dependency updates from europe-docker.pkg.dev/gardener-project/releases/gardener/ because these PRs are created by pipeline jobs.
+      "matchDatasources": ["docker"],
+      "matchPackagePatterns": ["europe-docker\\.pkg\\.dev\/gardener-project\/releases\/gardener\/.+"],
+      "matchFileNames": ["imagevector\/images\\.yaml"],
+      "enabled": false
```

Comment:
I would rather vote to get rid of these duplicative pipeline jobs and let Renovate take over completely. WDYT?

Reply:
Fair point.

@oliver-goetz Do you know how to achieve this? Removing this extra configuration in renovate is simply, but it will most likely lead to duplicate pull requests by concourse and renovate. I am not sure how to disable the concourse part.

Reply:
I think we have to adapt our pipeline configuration in the `refs/meta/ci` branch and remove the section that is responsible for these upgrade PRs.

Reply:
Is it [this entire section](https://github.com/gardener/gardener/blob/7b826d22c7cdf86af439340840d4be0c9c66fbdf/branch.cfg#L74-L91), which could be removed then? 
If we remove it, we should also remove [the section where we exclude `github.com/gardener/*`](https://github.com/gardener/gardener/blob/8f03dfcaaddafe8b58e39592c907852e8f2cffac/.github/renovate.json5#L77-L82) from renovate config.

Reply:
Yes, I think so @oliver-goetz

Reply:
Addressed for renovate in https://github.com/gardener/gardener/pull/9370/commits/24b62da67c7b8dd7d9c4ea4e6d063c4257ed1f2e.

Reply:
Removed the golang gardener component section as well with https://github.com/gardener/gardener/pull/9370/commits/13f461fa37839ca7f71bed22f5e8de9f016906b4.